build.gradle

```diff
@@ -251,7 +250,7 @@ gatlingRun.mustRunAfter(dockerTest)
 gatlingRun.dependsOn(startDocGenServerForGatling)
 gatlingRun.finalizedBy(stopDocGenServerForGatling)
 
-check.dependsOn(jacocoTestCoverageVerification, dockerTest, gatlingRun)
+//check.dependsOn(jacocoTestCoverageVerification, dockerTest, gatlingRun)
```

Comment:
Could you write a comment above this change explaining why it is not recommended doing this?

Reply:
Uploaded by mistake. Already fixed.